lib/options.js

```diff
@@ -35,13 +35,26 @@ function loadOptions(fileName, key, options) {
         return false;
       }
     }
+    if (overrides.extends) {
+      var shareableConfigPath = path.dirname(resolve
+        .sync('min-wd-config-' + overrides.extends, {
+          basedir: process.cwd()
```

Comment:
I'm using the current working directory as the `basedir` to allow for testing.

Reply:
👍